src/modules/builder/editor/modules/basic/BasicLayout.tsx

```diff
@@ -6,6 +6,7 @@ import BasicPanel from './components/BasicPanel';
 const BasicLayout = () => {
   const [activeTab, setActiveTab] = React.useState(0);
   const { set: onChangeText, values: basicTabs } = useBasicDetails();
+  const tabTitles = ['Contacts', 'Links', 'About'];
```

Comment:
move this constant outside of the function

Reply:
This is done. Moved the line outside function